base/remote/R/remote.copy.to.R

```diff
@@ -28,8 +28,8 @@ remote.copy.to <- function(host, src, dst, options = NULL, delete = FALSE, stder
   if (is.localhost(host)) {
     args <- c(args, src, dst)
   } else {
-    if (is.null(host$name) || host$name == "") {
-      PEcAn.logger::logger.severe("`name` parameter in the `host` object is NULL or empty : Try passing a valid host object")
+    if (is.null(host$name)) {
+      PEcAn.logger::logger.severe("`name` parameter in the `host` object is NULL : Try passing a valid host object")
```

Comment:
OK, so the changes in PEcAn.remote that you seem to have accidently reverted don't seem to be getting fixed automatically, so you might need to re-do these fixes manually

Reply:
done